### DISCUSSION TITLE
Support keyboard based filtering when searchable flag is set to false

### DISCUSSION THREAD

**Discussion by jochienabuurs** [mskocik/svelecte#247] [Ideas]
When looking at the minimal example shown [here](https://svelecte-v5.vercel.app/), setting "searchable" to false disables any keyboard based filtering of results.

It would be nice that when setting searchable to false, the select would behave as a native HTML select, allowing keyboard based filtering.

Is this something you'd consider adding?
Or maybe it is already possible but am I missing some configuration?

**Comment by mskocik** [comment]
Why would you want to do that, when you have `searchable` disabled? I am not considering this feature, although it's doable. But in general it's just adding code complexity with very little added value.

**Comment by jochienabuurs** [comment]
To make the input behave like a native html select more.


Op di 20 aug 2024 om 14:55 schreef Martin Skočík ***@***.***>:
